python_tests/test_python_frontend.py

```diff
@@ -1432,8 +1432,27 @@ def fusion_func(fd: FusionDefinition):
 
             nvf_stride = nvf_out[0].stride()
             sorted_stride = list(nvf_stride)
+            rank = len(nvf_stride)
+            for idx, axis in enumerate(perm):
+                sorted_stride[rank - 1 - axis] = nvf_stride[idx]
+            self.assertTrue(sorted(sorted_stride, reverse=True) == sorted_stride)
+
+            # testing stride_order in set
+            def fusion_set_func(fd: FusionDefinition):
+                t0 = fd.from_pytorch(inputs[0])
+                c0 = fd.define_scalar(3.0)
+                t1 = fd.ops.add(t0, c0)
+                t2 = fd.ops.set(t1, perm)
+                fd.add_output(t2)
+
+            nvf_out, _ = self.exec_nvfuser(fusion_set_func, inputs)
+            self.assertEqual(eager_out, nvf_out[0])
+
+            nvf_stride = nvf_out[0].stride()
+            sorted_stride = list(nvf_stride)
+            rank = len(nvf_stride)
             for idx, axis in enumerate(perm):
-                sorted_stride[axis] = nvf_stride[idx]
+                sorted_stride[rank - 1 - axis] = nvf_stride[idx]
```

Comment:
cc'ing @kevinstephano this was what I was talking about the updated `stride_order` semantics :laughing: Hope you don't mint it too much.